tools/test/h5ls/CMakeTests.cmake

```
@@ -403,12 +478,10 @@ if (${${HDF_PREFIX}_HAVE_COMPLEX_NUMBERS})
   # for example, rather than "complex number of native float".
   if (H5_WORDS_BIGENDIAN)
     ADD_H5_TEST (tcomplex_be RESULT_CODE 0 -w80 -v tcomplex_be.h5)
-    ADD_H5_TEST (tcomplex_be_nosupport RESULT_CODE 0 -w80 -v tcomplex_be.h5)
-    set_tests_properties (H5LS-tcomplex_be_nosupport PROPERTIES WILL_FAIL "true")
+    ADD_H5_TEST (tcomplex_be_nosupport RESULT_CODE 0 -w80 -v tcomplex_be.h5 ILL_FAIL)
```

```suggestion
    ADD_H5_TEST (tcomplex_be_nosupport RESULT_CODE 0 -w80 -v tcomplex_be.h5 WILL_FAIL)
```

Last change and this looks good